.github/workflows/android.yml

```diff
@@ -7,9 +7,9 @@ on:
   #   branches: 
   #   - main
     
-  pull_request:
+  push:
     branches:
-      - main
+      - improveSearchResults
```

Comment:
You will need to add dev branch as well.

Reply:
Hello @Akhror android.yml file is already updated  on Development. There are four files which is already updated on development so it creates conflict. so I will not merge in Development.